Modix.Services/Moderation/ModerationService.cs

```diff
@@ -524,14 +524,16 @@ public Task<IReadOnlyCollection<ModerationActionSummary>> SearchModerationAction
 
         public async Task<bool> DoesModeratorOutrankUserAsync(ulong guildId, ulong moderatorId, ulong subjectId)
         {
-            var moderator = await UserService.GetGuildUserAsync(guildId, moderatorId);
+            var guild = await DiscordClient.GetGuildAsync(guildId);
```

Comment:
`IUserService` abstracts this for a reason, so we can call `TrackUserAsync()` for both of these users and ensure that we have a record of them in the database, before attempting to create an `Infraction` record that will throw an exception if we don't.

If you're worried that we're going to be doing a duplicate `async` query to get the `IGuild`, don't. `DiscordClient` is actually a `DiscordSocketClient` abstracted to an `IDiscordClient`, so `GetGuildAsync()` is a 100% synchronous call.